.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 coverage
 .DS_Store
+package-lock.json
```

Comment:
This might be a relevant twitter thread https://twitter.com/maybekatz/status/869819920611016705